website/docs/d/service_health.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: "consul"
+page_title: "Consul: consul_service_health"
+sidebar_current: "docs-consul-data-source-health"
+description: |-
+  Filter service instances based on health status
+---
+
+# consul_service_health
+
+`consul_service_health` can be used to get the list of the instances that
+are currently healthy, according to their associated  health-checks.
+The result includes the list of service instances, the node associated to each
+instance and its health-checks.
```

Comment:
I think we should add a few lines here about this being a resource that will likely change frequently as service health changes (theoretically) quite a lot, and that you can expect different results on a frequent basis.